### DISCUSSION TITLE
Select data

### DISCUSSION THREAD

**Discussion by JanSkoda** [adminerevo/adminerevo#103] [General]
Why did the "select data" option disappear from the left column with the list of tables ? I was used to use it in Adminer, but here it is not. Or is there any possibility where to enable this feature ? Now i must select table at first, and then change "card" to "select data"

**Comment by LionelLaffineur** [comment]
Dear @JanSkoda, if I properly understand what you are talking about, that link has been replaced by the icon on left of the table name to spare space, but the functionality remains the same.  Regards, L.

**Comment by JanSkoda** [comment]
I'm so sorry I didn't notice. I didn't get the meaning of the icon at all. So consider my question unnecessary and excuse me.

**Reply by LionelLaffineur** [reply]
No reason to be sorry !  Happy to count you as a user of Adminer !
